packages/server/graphql/mutations/helpers/isStartMeetingLocked.ts

```diff
@@ -1,12 +1,30 @@
+import isValid from '../../../graphql/isValid'
+import getTeamsByOrgIds from '../../../postgres/queries/getTeamsByOrgIds'
 import {DataLoaderWorker} from '../../graphql'
 
 const isStartMeetingLocked = async (teamId: string, dataLoader: DataLoaderWorker) => {
   const team = await dataLoader.get('teams').loadNonNull(teamId)
-  const {isPaid, lockMessageHTML} = team
-  if (isPaid) return null
-  return lockMessageHTML
-    ? 'Wow, you’re determined to use Parabol! That’s awesome! Do you want to keep sneaking over the gate, or walk through the door with our Sales team?'
-    : 'Sorry! We are unable to start your meeting because your team has an overdue payment'
+  const {lockMessageHTML, orgId, tier} = team
+  const orgTeams = await getTeamsByOrgIds([orgId])
+  const anyOrgTeamIsLocked = orgTeams.some((team) => !team.isPaid)
+  if (tier !== 'personal' && anyOrgTeamIsLocked) {
+    // if this team was manually locked, be mean because they called this by hiding the modal
+    return lockMessageHTML
+      ? 'Wow, you’re determined to use Parabol! That’s awesome! Do you want to keep sneaking over the gate, or walk through the door with our Sales team?'
+      : 'Sorry! We are unable to start your meeting because your organization has an overdue payment'
+  }
+  // if this team wasn't manually locked, see if any of its members are on locked teams
+  const teamMembers = await dataLoader.get('teamMembersByTeamId').load(teamId)
+  const userIds = teamMembers.map(({userId}) => userId)
+  const allTeamMembers = (await dataLoader.get('teamMembersByUserId').loadMany(userIds))
+    .filter(isValid)
+    .flat()
+  const teamIds = [...new Set(allTeamMembers.map(({teamId}) => teamId))]
+  const allRelatedTeams = (await dataLoader.get('teams').loadMany(teamIds)).filter(isValid)
+  const anyTeamIsLocked = allRelatedTeams.some((team) => team.isPaid === false)
+  return anyTeamIsLocked
+    ? 'One or more of your team members is over the usage limit. Please contact sales'
```

Comment:
We could mention the team member(s) & which team(s) are locked in this message. That would be a helpful UX, but it increases the likelihood that they remove that team member / delete the team to get around the rules.

Reply:
heh, i was thinking the same thing! we want them to solve the problem with money, so the less info the better